## About Me

![Ábel](https://raw.githubusercontent.com/vertesy/vertesy.github.io/master/images/pics_AV/profile%202016-01-22_17.49.19.edit.small.jpg)

I am [Ábel Vértesy](https://github.com/vertesy), a postdoc in the [Knoblich Lab](https://www.imba.oeaw.ac.at/research/juergen-knoblich/) at [IMBA - Institute for Molecular Biotechnology](https://www.imba.oeaw.ac.at/?L=EN) of the Austrian Academy of Sciences in Vienna. I did my PhD in [Alexander van Oudenaarden's lab](http://www.hubrecht.eu/onderzoekers/van-oudenaarden-group/) @ the [Hubrecht Institute](http://www.hubrecht.eu), in Utrecht, The Netherlands. I currently focus on various single-cell projects in brain organoids. During my PhD, I was focusing on data analysis in different single-cell genomics and spatial transcriptomics projects. This is my personal website summarizing some of my science related activities. 


## Science

My papers are listed on [ORCID](https://orcid.org/0000-0001-6075-5702), on [Google Scholar](https://scholar.google.nl/citations?user=EugkUHAAAAAJ&hl=nl&oi=ao) and on [Pubmed](https://www.ncbi.nlm.nih.gov/pubmed/?term=Vertesy+A%5BAuthor%5D).

Preprints of my manuscripts are accessible on [biorxiv](https://www.biorxiv.org/search/author1%3AVertesy%20numresults%3A10%20sort%3Arelevance-rank%20format_result%3Astandard), or accesible on request.



## Teaching

I designed and held two summer courses in the July-Aug/2018 in Accra, at the University of Ghana.

1. [Single-cell genomics](https://vertesy.github.io/Single-Cell-Omics-Course/) (1 course)
2. [Exploratory-data-analysis-in-R](https://vertesy.github.io/Exploratory-data-analysis-in-R/) (2 courses)



## Software

During my work I also wrote several R-packages:

1. [**MarkdownReports**](https://vertesy.github.io/MarkdownReports), an R function library to create scientific figures and reports easily.
1. [**MarkdownHelpers**](https://github.com/vertesy/MarkdownHelpers),  Helper functions (control and parsing) for `MarkdownReports` and `ggExpres`.
3. [**ggExpress**](https://vertesy.github.io/ggExpress),  create, annotate and export figures with minimal code and effort, using ggplot.
4. [**Stringendo**](https://vertesy.github.io/Stringendo), with string parsing functionalities for generating plotnames, filenames and path.
5. [**ReadWriter**](https://vertesy.github.io/ReadWriter),  to read and write tabular data with minimal code and effort.
2. [**CodeAndRoll2**](https://vertesy.github.io/CodeAndRoll2), a utility function library for R.
3. [**Seurat.utils**](https://vertesy.github.io/Seurat.utils), various utility functions for single-cell analysis with Seurat v3.
4. [**RoxygenReady**](https://vertesy.github.io/RoxygenReady) to prepare your R function library to be compiled by Roxygen into an R package.
7. [**DatabaseLinke.R**](https://vertesy.github.io/DatabaseLinke.R), functions for easier lookup of genes & proteins in various databases via your browser.
10. [**DataInCode**](https://vertesy.github.io/DataInCode) to store (smaller) data objects in your R code - make your script fully reproducable yet easily sharable. 
7. [**Seurat.multicore**](https://vertesy.github.io/Seurat.multicore), multicore and utility functions mostly for Seurat 2, using `doMC` / `foreach` packages.





#### [A older, short presentation on the first tools I wrote](https://docs.google.com/presentation/d/1shVYLLhLhbcIIWNbiAgrugtqeZh_VRgKiLr_BvgAWEc/embed?start=false&loop=false&delayms=5000#slide=id.p4) :

[![CoverImage](https://snag.gy/BMPxIl.jpg)](https://docs.google.com/presentation/d/1shVYLLhLhbcIIWNbiAgrugtqeZh_VRgKiLr_BvgAWEc/embed?start=false&loop=false&delayms=5000#slide=id.p4)

